### DISCUSSION TITLE
www domain working but root domain forwarding not

### DISCUSSION THREAD

**Discussion by Ash3156** [vercel/community#4603] [Help] [answered]
### Summary

Hello! I added my domains https://ax.al and https://www.ax.al to Vercel, with ax.al redirecting (308) to www.ax.al - the Domain page says DNS records are set up correctly and I should be good. However, while www.ax.al is working fine, I keep getting a long white screen eventually resulting in ERR_CONNECTION_TIMED_OUT. Furthermore, running dig ax.al +short gives me "185.26.105.244", which does not match up with Vercel.

### Steps to Reproduce

run dig ax.al +short (incorrect) and compare it with dig www.ax.al +short (working).

Domain registrar DNS records:
![Screenshot 2023-10-24 at 11 20 47 PM](https://github.com/vercel/community/assets/65479301/3fcd2d86-3e0b-4653-9cd7-7e24afccc7f3)

Vercel domain status:
![Screenshot 2023-10-24 at 11 30 06 PM](https://github.com/vercel/community/assets/65479301/7990c851-bbb3-4ca0-80d1-aff2dace4317)

**Comment by amyegan** [comment]
Hey there! It looks like this may have been because of the normal delay in DNS propagation. When I check the [A record for ax.al](https://www.whatsmydns.net/#A/ax.al) and the [CNAME for www.ax.al](https://www.whatsmydns.net/#CNAME/www.ax.al), I get back the expected values.

Are you still having trouble with this domain?

**Comment by Ash3156** [comment]
Hey Amy, I still can't access ax.al despite the records propagating (www.ax.al works). Are you able to?

Reaching out to my registrar as well - the dig ax.al +short gives me 185.26.105.244 (not Vercel's 76.76.21.21 like the whatsmydns link you sent says), which seems to be one of their redirect servers so may be an issue on their end.

**Reply by amyegan** [reply]
I also tried `dig ax.al +short`  and it returns `76.76.21.21` just like whatsmydns. Can you try from another computer or network?

**Reply by Ash3156** [reply]
It works fine on a different network (same computer), strange since the records look to be populated everywhere? I cleared my cache + restarted computer and its still giving me issues on this particular wifi network. Do you know what the issue might be?

**Comment by Ash3156** [comment, marked as answer]
Solution: on my computer, running `sudo killall -HUP mDNSResponder` was the only thing that worked (clearing DNS cache).
